test/integration/kubernetes_deploy_test.rb

```diff
@@ -917,6 +919,19 @@ def test_cronjobs_can_be_deployed
     cronjobs.assert_cronjob_present("my-cronjob")
   end
 
+  def test_jobs_can_be_successful
+    assert_deploy_success(deploy_fixtures("jobs"))
+  end
+
+  def test_jobs_can_fail
+    fixtures = deploy_fixtures("jobs") do |f|
+      spec = f["job.yml"]["Job"].first["spec"]["template"]["spec"]
+      spec["backoffLimit"] = 1
+      spec["containers"].first["args"] = %w(FAKE)
+    end
+    assert_deploy_failure(fixtures)
```

Comment:
What does this look like? What status are we printing? Is the failure information we're printing as helpful as it could be?